pygfx/cameras/_orthographic.py

```diff
@@ -33,29 +31,23 @@ def __repr__(self) -> str:
         )
 
     def set_viewport_size(self, width, height):
-        self._view_aspect = width / height
-
-    def update_projection_matrix(self):
-        # Get the reference width / height
+        self.width = float(width)
+        self.height = float(height)
         width = self.width / self.zoom
         height = self.height / self.zoom
-        # Increase eihter the width or height, depending on the view size
-        aspect = width / height
-        if aspect < self._view_aspect:
-            width *= self._view_aspect / aspect
-        else:
-            height *= aspect / self._view_aspect
```

Comment:
Why is this removed? The ortho camera is broken atm, because the content does not scale as the window is resized. Not sure if the missing of these lines are the only problem.

Reply:
Haha. Well. Remember this discussion? https://github.com/pygfx/pygfx/pull/53#pullrequestreview-457309496